server/sonar-process/src/main/java/org/sonar/process/ProcessEntryPoint.java

```diff
@@ -104,6 +105,13 @@ public void launch(Monitored mp) {
         Thread.sleep(20L);
       }
```

Comment:
this block could be in a method
(I should have done it already, missed/forgot to do it)